app/src/main/java/com/teamagam/gimelgimel/app/common/logging/LogWrapper.java

```diff
@@ -0,0 +1,43 @@
+package com.teamagam.gimelgimel.app.common.logging;
+
+/**
+ * Log wrapper class to abstract the use of loggers.
+ * Contains application-specific logging functionality.
+ */
+public interface LogWrapper {
+
+    void d(String message);
+    void d(String message, Throwable tr);
+
+    void e(String message);
+    void e(String message, Throwable tr);
+
+    void i(String message);
+    void i(String message, Throwable tr);
+
+    void v(String message);
+    void v(String message, Throwable tr);
+
+    void w(String message);
+    void w(String message, Throwable tr);
+
+    void userInteraction(String message);
+
+    void onCreate();
```

Comment:
make sure we didnt miss anthing. go over the life cycle of activities and fragments (onAttach)